OnboardPietroPieri/validation/validation.tsx

```diff
@@ -24,31 +25,64 @@ const isvalidateBirthDate = (birthDate: string) => {
   return selectedDate <= minimumDate;
 };
 
-const formValidate = (
-  password: string,
-  email: string,
-  birthDate: string,
-): boolean => {
-  if (!isvalidateBirthDate(birthDate)) {
+const isEmpty = (value: string): boolean => {
+  if (value === "") {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Quando vc vê ifs que retornam true ou false, é um sinal que você pode trabalhar com a própria expressão booleana. Nesse caso, é equivalente fazer:

```suggestion
  return value === ""
```

Se for uma string vazia, esa expressão já retorna true. Senão, já retorna false. Só que refletindo bem, talvez nem prcise de uma função pra isso né?